homeassistant/components/homematic/light.py

```diff
@@ -77,6 +77,9 @@ def effect(self):
 
     def turn_on(self, **kwargs):
         """Turn the light on and/or change color or color effect settings."""
+        if ATTR_TRANSITION in kwargs:
+            self._hmdevice.setValue('RAMP_TIME', kwargs[ATTR_TRANSITION])
```

Comment:
Apart from making `RAMP_TIME` a variable (`ATTR_RAMP_TIME = 'RAMP_TIME'` somewhere at the top) for the sake of consistency within HA everything seems fine to me. 👍